PySpark tpch fail to connect

[Workflow Run URL](https://github.com/coiled/benchmarks/actions/runs/6818965990)

```python
E           pyspark.errors.exceptions.connect.SparkConnectGrpcException: <_MultiThreadedRendezvous of RPC that terminated with: E           	status = StatusCode.UNAVAILABLE E           	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:3.144.72.177:15003: Failed to connect to remote host: FD Shutdown" E           	debug_error_string = "UNKNOWN:failed to connect to all addresses; last error: UNKNOWN: ipv4:3.144.72.177:15003: Failed to connect to remote host: FD Shutdown {grpc_status:14, created_time:"2023-11-10T01:18:52.697409071+00:00"}" E           >
```

--- a/tests/tpch/conftest.py
+++ b/tests/tpch/conftest.py
@@ -155,14 +155,6 @@ def cluster_spec(scale):
         idle_timeout="1h",
         wait_for_workers=True,
         scheduler_vm_types=["m6i.xlarge"],
-        backend_options={
-            "ingress": [
-                {
-                    "ports": [443, 8786, 8787, 7077, 8080, 4040, 15002],
-                    "cidr": "0.0.0.0/0",
-                },
-            ],
-        },
     )
     if scale == 10:
         return {
@@ -260,10 +252,7 @@ def spark_setup(cluster, local):
 
         spark = SparkSession.builder.master("local[*]").getOrCreate()
     else:
-        from coiled.spark import get_spark
-
-        with cluster.get_client() as client:
-            spark = get_spark(client)
+        spark = cluster.get_spark()
 
     # warm start
     from pyspark.sql import Row

